src/components/Checkbox/Checkbox.js

```diff
@@ -43,6 +45,10 @@ export const Checkbox = ({
     overrideDefaultChecked = false;
   }
 
+  useEffect(() => {
+    inputRef.current.indeterminate = indeterminate;
```

Comment:
Hi @fatihict ,
Thanks for this improvement!, I have only one question - 
This will render only on the initial render, don't you think it will be better if `indeterminate` will be given to the dependency array, so it could be controlled by the consumer?

Thanks

Reply:
Hey @orrgottlieb,
Fair point, I missed that. Fixed your comment in a new commit.

Reply:
thanks! 🤩